src/CoursePageResources/CoursePageResources.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import { Button } from '@edx/paragon';
+import CoursePage from '../course-page/CoursePage';
+
+function CoursePageResources() {
+  return (
+    <main>
+      <div className="container-fluid">
+        <div className="container-fluid-container d-flex justify-content-between align-items-center border-bottom">
+          <h1 className="container-fluid-heading mt-3">Pages & Resources</h1>
+          <Button className="container-fluid-button font-weight-bold rounded my-3">
+            View Live
+          </Button>
+        </div>
+        <div className="container-fluid-sub-container">
+          <h3 className="container-fluid-sub-container-heading mt-3">
+            Course pages
+          </h3>
+          <div className="row">
+            <div className="col">
+              <CoursePage
+                title="Discussion"
+                configure={false}
+                enabledButton
+                coursePageStatus={false}
+                description="Encourage Participation and engagement in your course with discussion forums"
+              />
```

Comment:
This should not be hard-coded, it will be constructed based on data from the API.

Reply:
I assume you mean `cousePageStatus`.

Reply:
Note: As I mentioned above, these tiles should not be hard-coded, they will be returned by an API. So you did not need to move all this to the translation file. i.e. Remove all `<CoursePageConfigTile` objects from here and have this list be provided by the parent element for `CoursePageResources`